bedrock/settings/base.py

```diff
@@ -750,7 +750,8 @@ def get_app_name(hostname):
 EMAIL_HOST_PASSWORD = config("EMAIL_HOST_PASSWORD", default="")
 
 # Google Analytics
```

Comment:
Nit: suggest grouping both `GOOGLE_ANALYTICS_ID` and `GTM_CONTAINER_ID` together under this heading, so that we can more easily explain which site uses which setting? Kinda conscious that over time this type of thing might grow more confusing, unless we start to organise settings a bit more?

Reply:
Done - thanks!